robustcontrol/utils.py

```diff
@@ -1721,6 +1721,24 @@ def feedback_mimo(G, K=None, positive=False):
 #                                Chapter 4                                    #
 ###############################################################################
 
+def checkvalid(G, z):
```

Comment:
This is not a very intuitive name. Consider that this actually returns the minimum singular value evaluated at particular frequencies, so perhaps should be called sigma_min or min_singular? And perhaps that suggests we can generalise to return the minimum or the maximum - something that we need frequently.

Reply:
Changed to min_max_sigma.